pkg/processor/runtime/python/py/requirements/python3_11.txt

```diff
@@ -1,2 +1 @@
 -r common.txt
-msgpack==1.0.8
```

Comment:
was it removed become it exists on `common.txt`?

Reply:
@liranbg fixed that. In devel we aligned everything and moved to common.txt. Won't do this for 1.13.x, so each python version has its own version of msgpack